src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -285,6 +285,13 @@ def _check_create_job_wait_canceled() -> bool:
 
 
 @cli_bundle.command(name="gui-submit")
+@click.option(
+    "-p",
+    "--parameter",
+    multiple=True,
+    callback=validate_parameters,
+    help='The values for the job template\'s parameters. Can be provided as key-value pairs, inline JSON strings, or as paths to a JSON or YAML document. If provided more than once, the values are combined in the order that they appear. Examples: --parameter MyParam=5 -p file://parameter_file.json -p \'{"MyParam": "5"}\'',
```

Comment:
It'd be helpful to document how these parameters interact with the GUI. If a parameter is set, does it show up in the GUI? If a value is updated in the GUI, does the value in the command or the value in the GUI get used? I think this basically sets the default values in the GUI.